### DISCUSSION TITLE
upstream vs service

### DISCUSSION THREAD

**Discussion by tirelibirefe** [apache/apisix#6772] [Q&A]
Hello,
I'm a newbie for apisix and trying to understand the concept. 
Yes, I've read the documents but I didn't understand.
When I install httpbin, is its service an upstream or a service in Apisix terminology? If it is a service, what is upstream? If it is an upstream, what is service?

Thanks & Regards

**Comment by guitu168** [comment]
There is a one-to-one correspondence between services and upstream. To be precise, httpbin itself is a service, but it acts as an upstream of Apache APISIX.
